src/qibosoq/rfsoc_server.py

```diff
@@ -66,20 +88,23 @@ def handle(self):
         * Executes qick program
         * Return results
         """
-        # print a log message when receive a connection
-        logger.debug("Got connection from %s", self.client_address)
 
         # set the server in non-blocking mode
         self.server.socket.setblocking(False)
 
         try:
+            logger.debug("Receiving data")
             data = self.receive_command()
+            logger.debug("Executing program")
             results = self.execute_program(data)
-        except Exception as exception:
+        except Exception as exception:  # pylint: disable=bare-except, broad-exception-caught
```

Comment:
Do you really need such a broad exception?

Reply:
I don't know everything of socket and qick libraries, but I want that whatever error get raised, by whatever library:
* the client gets to see it
* the server does not stop
Not sure if there is a better way